operation.go

```diff
@@ -41,6 +42,30 @@ type DetailedTransaction struct {
 	Hash      string               `json:"hash"`
 }
 
+// GetTransactionByTx gets transaction details from a specific Tx
+// confirmed => "true"; failed => "false"; pending => "";
+func (c *TZKT) GetTransactionStatusByTx(hash string) (string, error) {
```

Comment:
```suggestion
func (c *TZKT) GetTransactionStatusByTx(hash string) (*bool, error) {
```
return true if confirm
false if failed
nil if pending (not found)